interfaces/builtin/camera.go

```diff
@@ -48,12 +51,13 @@ const cameraConnectedPlugAppArmor = `
 /run/udev/data/c81:[0-9]* r, # video4linux (/dev/video*, etc)
 /run/udev/data/+usb:* r,
 /sys/class/video4linux/ r,
-/sys/devices/pci**/usb*/**/video4linux/** r,
+/sys/devices/**/video4linux/** r,
 `
 
 var cameraConnectedPlugUDev = []string{
 	`KERNEL=="video[0-9]*"`,
 	`KERNEL=="vchiq"`,
+	`KERNEL=="io[0-9]*"`,
```

Comment:
This change appears to be triggering a unit test failure:

```
----------------------------------------------------------------------
FAIL: camera_test.go:83: CameraInterfaceSuite.TestUDevSpec

camera_test.go:86:
    c.Assert(spec.Snippets(), HasLen, 3)
... obtained []string = []string{"# camera\nKERNEL==\"io[0-9]*\", TAG+=\"snap_consumer_app\"", "# camera\nKERNEL==\"vchiq\", TAG+=\"snap_consumer_app\"", "# camera\nKERNEL==\"video[0-9]*\", TAG+=\"snap_consumer_app\"", "TAG==\"snap_consumer_app\", RUN+=\"/usr/lib/snapd/snap-device-helper $env{ACTION} snap_consumer_app $devpath $major:$minor\""}
... n int = 3
```

From the look of it, you can just change the `HasLen` assertion to 4, and add an extra `testutils.Contains` assertion for the new rule you've added.